services/order.py

```diff
@@ -0,0 +1,28 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from django.db import transaction
+
+from db.models import Order, Ticket, User, MovieSession
+
+
+def create_order(tickets: list[dict], username: str, date: datetime = None) -> Order:
+    with transaction.atomic():
+        user = User.objects.get(username=username)
```

Comment:
The `create_order` function uses `User.objects.get(username=username)`, which will raise a `DoesNotExist` exception if no user with the given username is found. Consider handling this exception to improve robustness.